### PR TITLE
closes #2

### LINKED ISSUE #2: Relative timestamps should automatically update via js to countdown
The `{time_end:relative}`/`{time_start:relative}` should update via js to count-down without needing toe refresh the page.

Format;
< 5 months, 16 days, 7 hours, 55 minutes, 59 seconds>

### PR DESCRIPTION
**Pull request by ticktackk**

### CODE REVIEW

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        {
+            if (typeof value === 'string')
+            {
+                value = parseInt(value) || 0;
+            }
+
+            if (typeof phrase !== 'string' || !phrase)
+            {
+                console.error('Invalid phrase provided.');
+                clearInterval(this.timer);
```
Comment (Xon) [inline upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: The other `return` doesn't trigger this, is this expected?

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        getDatePart: function (value, phrase)
+        {
+            if (typeof value === 'string')
+            {
+                value = parseInt(value) || 0;
+            }
+
+            if (typeof phrase !== 'string' || !phrase)
+            {
+                console.error('Invalid phrase provided.');
```
Comment (Xon) [inline upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: Should log the phrase so there is some idea what is wrong

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                return false;
+            }
+
+            var translatedValue = XF.phrase('svNoticeTimeReplacables_' + phrase + (value > 1 ? 's' : ''), {
+                '{count}': value
+            }, null);
+
+            if (translatedValue === null)
+            {
+                console.error('Invalid phrase provided.');
```
Comment (Xon) [inline upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: Should log the phrase so there is some idea what is wrong

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            {
+                var $noticeContent = this.$target.closest('.notice-content'),
+                    $noticeDismissButton = $noticeContent.length() ? $noticeContent.find('.notice-dismiss') : null;
+
+                if ($noticeDismissButton.length)
+                {
+                    $noticeDismissButton.trigger('click');
+                }
+
+                this.$target.text();
```
Comment (Xon) [inline upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js:172]: Is this debug code to be removed? Or an attempt to blank the target text?

**Review by Xon — CHANGES_REQUESTED** [review]
Not sure if `moment.min.js.map` should be included?

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js (hunk 3 above)
Comment (ticktackk) [reply upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: Should log now.

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js (hunk 2 above)
Comment (ticktackk) [reply upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: Should log the phrase now.

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js (hunk 1 above)
Comment (ticktackk) [reply upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js]: Should return now.

File: upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js (hunk 4 above)
Comment (ticktackk) [reply upload/src/addons/SV/NoticeTimeReplacable/_files/js/sv/notice-time-replacable/core.js:172]: That debug code should be gone now.

File: upload/src/addons/SV/NoticeTimeReplacable/XF/NoticeList.php [hunk 5]
```diff
@@ -162,9 +162,11 @@ public function getRelativeDate(\DateTime $now, \DateTime $other)
             ]);
         }
 
+        $language = \XF::language();
         return '<span class="time-notice" data-xf-init="sv-notice-time-replacable--relative-timestamp" ' .
-            'data-now="' . \XF::escapeString($now->getTimestamp()) . '" ' .
             'data-timestamp="' . \XF::escapeString($other->getTimestamp()) . '" ' .
+            'data-date-format="' . $language->date_format . '" ' .
+            'data-time-format="' . $language->time_format . '" ' .
```
Comment (Xon) [inline upload/src/addons/SV/NoticeTimeReplacable/XF/NoticeList.php]: Any reason not to use ` \XF::escapeString($language->time_format)` here ?

**Pull request merged**